### PR TITLE
Create barbara-budrich.csl

### PR DESCRIPTION
**Pull request by jg-schwarz**
Added a style for German Publisher Barbara Budrich (https://budrich.de/)

### CODE REVIEW

File: barbara-budrich.csl [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  </macro>
+  <macro name="online-url">
+    <group delimiter=" ">
+      <text variable="URL"/>
+      <date form="numeric" variable="accessed" prefix="[Zugriff: " suffix="]"/>
+    </group>
+  </macro>
+  <macro name="doi">
+    <group delimiter=" ">
+      <text variable="DOI" prefix="DOI: " suffix=""/>
```
Comment (bwiernik) [inline barbara-budrich.csl:102]:
```suggestion
      <text variable="DOI" prefix="https://doi.org/"/>
```

Their books use a mix of DOI formatting, so let's go with the modern recommended style from CrossRef

File: barbara-budrich.csl [hunk 2]
```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="de-DE">
```
Comment (bwiernik) [inline barbara-budrich.csl:2]:
```suggestion
<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
```

They publish mostly in German, but do also have English publications, so let's make the style support both

File: barbara-budrich.csl [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    <category field="social_science"/>
+    <category field="humanities"/>
+    <summary>This citation style for books and edited volumes of the publisher Barbar Budrich is based on the 'Style Sheet' for authors and editors from 2024 which is unfortunately not available from the publisher's website.</summary>
+    <updated>2024-06-13T17:28:01+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de">
+    <terms>
+      <term name="et-al">et al.</term>
+      <term name="online">Online unter</term>
```
Comment (bwiernik) [inline barbara-budrich.csl:24]:
```suggestion
      <term name="online">Online unter</term>
      <term name="accessed">Zugriff</term>
```

File: barbara-budrich.csl [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+        </group>
+        <choose>
+          <if type="book" match="any">
+            <group delimiter=". ">
+              <text macro="edition"/>
+              <text macro="publication-place-publisher"/>
+              <text macro="collection-title-number"/>
+              <group delimiter=". ">
+                <text macro="doi"/>
+              </group>
```
Comment (bwiernik) [inline barbara-budrich.csl:156]:
```suggestion
              <text macro="doi"/>
```

File: barbara-budrich.csl [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+            </group>
+          </else-if>
+          <else-if type="chapter" match="any">
+            <group delimiter=", ">
+              <group delimiter=". ">
+                <text macro="editor-container-title"/>
+                <text macro="edition"/>
+                <text macro="publication-place-publisher"/>
```
Comment (bwiernik) [inline barbara-budrich.csl:176]:
```suggestion
                <text macro="publication-place-publisher"/>
                <text macro="doi"/>
```

File: barbara-budrich.csl [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+                </if>
+              </choose>
+              <text macro="publication-place-publisher"/>
+              <text macro="page-range"/>
+              <text macro="collection-title-number"/>
+              <text macro="online-url"/>
+              <group delimiter=". ">
+                <text macro="page-range"/>
+                <text macro="doi"/>
+              </group>
```
Comment (bwiernik) [inline barbara-budrich.csl:194]:
```suggestion
              <text macro="page-range"/>
              <text macro="doi"/>
```

**Review by bwiernik — APPROVED** [review]
Thanks! Made a few comments and suggestions

**Comment by jg-schwarz** [comment]
Thank you for these improvements!

**Pull request merged**